### DISCUSSION TITLE
paypal payment do not work on testing mode

### DISCUSSION THREAD

**Discussion by benrandja-akram** [swellstores/community#172] [Help]
I've connected swell to my sandbox account. but it seems there a bug or something on your side,paypal popup opens with an error, i'm pretty sure that my sandbox env works fine, 
here is a screenshot: 

![image](https://user-images.githubusercontent.com/32803395/223895383-ba9650b0-d31f-4a5b-ad9f-882cd44f3965.png)

**Comment by mustafaswell** [comment]
@benrandja-akram : are you accessing this modal from **Swell > Payments > PayPal Express > Settings**? Can you use [Loom](https://www.loom.com/) to send a recording on your flow?

Thanks!
